### PR TITLE
Hometab updates

### PR DESCRIPTION
**Pull request by joeizang**
Update the hometab of IDE based on designs on excalidraw from @LianaHus

### CODE REVIEW

**Comment by LianaHus** [comment]
- [ ]  New file should create a file in current workspace
- [ ] some more spacing for title of File section (Exlpore...)
- [ ] add space after recent section and remove from the top of it
- [ ] slides should have more height
- [ ]  <i class="fa-regular fa-desktop"></i> for the Connect to Local File system button
- [ ] Alert alert-danger for alerts on the status bar and danger text color for icon and the text
- [ ] Alert window height should be adjusted
- [ ] align sections on status bar
- [ ] increase a margin-bottom for toaster
- [ ] If the workspace is not a github repo the text "not a git repo" can become a btn "init as git repo"
- [ ] all status bar sections should have tooltips
- [ ] fo**r next iteration:** Plugins from the selected list should be able to register a component/html el and a callback.

**Review by ryestew — COMMENTED** [review]
Exciting PR!!
I do have some comments:
the visited link in the red is a bit dark:
image

This is a minor issue - so it can be merged as is.

A more important point is that because LearnEth is gone, there should be links to it from one of the scrolling boxes.  This request is outside of this PR.

**Comment by vermouth22** [comment]
Looks great! However one fundamental issue, take note the attached image is the res seen on a 13 inch Mac M1 laptop. This shows even less real estate than the existing IDE hometab. 
1. Something should be done on progressive resolution changes per machine.
2. Tagline for Remix should say: "Learn. Explore. Create." because "Native IDE" we decided a while ago, has no meaning
3. Links below tagline should be "Website   Remix Desktop    Documentation" ... In this iteration there are two links for Desktop
4. Tagline below that (above Start Coding)... "Explore. Prototype. Create" should be removed
5. "Recent workspaces", should be rendered in U/L so it should read "Recent Workspaces"
6. "Connect to local filesystem" should also be U/L so it should read "Connect to Local Filesystem"

image

**Comment by vermouth22** [comment]
@joeizang Getting closer. FYI, there's still a bit of the feature panel cut off horizontally. And, now somehow need to lift up the Featured Plugins list...
image

**Comment by vermouth22** [comment]
1440 x 900 13" Macbook default display as of now: 
image

Still a bit of overlap on that "Files" section with HTTPS. And, some touching up top with the Language button, but looking lots better. Also, do you guys think we should separate those social icons a bit more, or are they just bunched together that closely on this res?

**Comment by ryestew** [comment]
The link for remix desktop should be updated to this one:
https://github.com/remix-project-org/remix-desktop-insiders

**Review by ryestew — APPROVED** [review]

File: apps/remix-ide/src/app/components/preload.tsx [hunk 1]
```diff
@@ -31,8 +31,10 @@ export const Preload = (props: any) => {
   )
 
   function loadAppComponent() {
+    console.log('loading remix in the preloader')
```
Comment (LianaHus) [inline apps/remix-ide/src/app/components/preload.tsx:34]: please remove

File: apps/remix-ide/src/app/components/preload.tsx [hunk 2]
```diff
@@ -68,6 +70,7 @@ export const Preload = (props: any) => {
     if (fsLoaded) {
       console.log(fsLoaded.name + ' activated')
       _paq.push(['trackEvent', 'Storage', 'activate', fsLoaded.name])
+      console.log('loading remix in the preloader')
```
Comment (LianaHus) [inline apps/remix-ide/src/app/components/preload.tsx:73]: please remove

File: apps/remix-ide/src/app/components/preload.tsx [hunk 3]
```diff
@@ -31,8 +31,10 @@ export const Preload = (props: any) => {
   )
 
   function loadAppComponent() {
+    console.log('loading remix in the preloader')
     import('../../app')
       .then((AppComponent) => {
+        console.log('loading remix in the preloader', AppComponent)
```
Comment (LianaHus) [inline apps/remix-ide/src/app/components/preload.tsx:37]: please remove

File: libs/remix-ui/home-tab/src/lib/components/homeTabFile.tsx [hunk 4]
```diff
@@ -216,14 +216,7 @@ contract HelloWorld {
...
     <>
-      <ModalDialog
-        id="homeTab"
-        title={'Import from ' + state.modalInfo.title}
-        okLabel="Import"
-        hide={!state.showModalDialog}
-        handleHide={() => hideFullMessage()}
-        okFn={() => processLoading(state.modalInfo.title)}
-      >
+      <ModalDialog id="homeTab" title={'Import from ' + state.modalInfo.title} okLabel="Import" hide={!state.showModalDialog} handleHide={() => hideFullMessage()} okFn={() => processLoading(state.modalInfo.title)}>
```
Comment (LianaHus) [inline libs/remix-ui/home-tab/src/lib/components/homeTabFile.tsx:219]: this is too long to be a 1 line

File: libs/remix-ui/home-tab/src/lib/components/homeTabFile.tsx [hunk 5]
```diff
@@ -253,62 +246,11 @@ contract HelloWorld {
...
-              </button>
-            </CustomTooltip>
-          </div>
+        <div className="mb-3">
           {(state.recentWorkspaces[0] || state.recentWorkspaces[1] || state.recentWorkspaces[2]) && (
-            <div className="d-flex flex-column">
+            <div className="d-flex flex-column mb-5 remixui_recentworkspace">
               <label style={{ fontSize: '0.8rem' }} className="mt-3">
-                Recent workspaces
+              Recent Workspaces
```
Comment (LianaHus) [inline libs/remix-ui/home-tab/src/lib/components/homeTabFile.tsx:253]: please add the tab

File: apps/remix-ide/src/app/components/preload.tsx (hunk 1 above)
Comment (Aniket-Engg) [reply apps/remix-ide/src/app/components/preload.tsx:34]: this is removed in https://github.com/ethereum/remix-project/pull/4897

File: apps/remix-ide/src/app/components/preload.tsx (hunk 3 above)
Comment (Aniket-Engg) [reply apps/remix-ide/src/app/components/preload.tsx:37]: this is removed in https://github.com/ethereum/remix-project/pull/4897

File: libs/remix-ui/home-tab/src/lib/components/homeTabFile.tsx (hunk 4 above)
Comment (joeizang) [reply libs/remix-ui/home-tab/src/lib/components/homeTabFile.tsx:219]: ok

**Pull request merged**